src/model/utilities/computerPlayer.js

```diff
@@ -0,0 +1,24 @@
+const computerPlayer = (playerToExtend) => {
+  const returnedPlayer = playerToExtend;
+  returnedPlayer.questionAndAnswersToAnswer = {};
+
+  returnedPlayer.askQuestion = (showQuestionToPlayer, questionAndAnswers, handlePlayerAnswered) => {
+    returnedPlayer.questionAndAnswersToAnswer = questionAndAnswers;
+
+    showQuestionToPlayer(questionAndAnswers);
+    // eslint-disable-next-line func-names
+    setTimeout(function () {
+      returnedPlayer.answer();
+      handlePlayerAnswered();
+    }, 1000);
+  };
+  returnedPlayer.answer = () => {
+    const randomIndex = Math.floor(
+      Math.random() * returnedPlayer.questionAndAnswersToAnswer.answers.length,
+    );
+    returnedPlayer.randomAnswer = returnedPlayer.questionAndAnswersToAnswer.answers[randomIndex];
+  };
```

Comment:
W tym miejscu zachęcam do użycia mojej funkcji 'randomValueFromArray' :) Jak już ją dodaliśmy do Utilities to fajnie jakby ktoś inny też z niej korzystał :D

Reply:
Ok, celna uwaga.